corehq/apps/linked_domain/models.py

```diff
@@ -34,7 +35,8 @@ class DomainLink(models.Model):
     deleted = models.BooleanField(default=False)
 
     # used for linking across remote instances of HQ
-    remote_base_url = models.CharField(max_length=255, null=True, blank=True)
+    remote_base_url = models.CharField(max_length=255, null=True, blank=True,
+                                       help_text=_("should be the full link with the trailing /"))
```

Comment:
An example would be helpful, like in the help text above, then people can mimic the example rather than think about the description means.